posts/2024-04-15_gno-tokyo/README.md

```diff
@@ -0,0 +1,89 @@
+---
+publication_date: 2024-04-15T00:00:00Z
+slug: gno-tokyo
+tags: [meetup, tokyo, community]
+authors: [leohhhn]
+---
+    
+# Intro to Gno Recap - Tokyo 2024
+[![Banner](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/thumbs/banner.png)](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/banner.png)
+
+Last week, we hosted the first ever Gno.land meetup in Tokyo, Japan with the
+event taking place at [Crypto Lounge GOX](https://cryptoloungegox.com/) 
+located in one of the city's most central points, Shinjuku City. We had 20+
+attendees from a variety of backgrounds from Web3, to developers, to traditional 
+business and fintech. The meetup, presented in Japanese, is part of our ongoing 
+initiative to host regional, in-person opportunities for people around the world
+to get to know Gno.land.
+
+With this regional focus in mind, we hosted in March a 
+[local meetup in Korea](https://medium.com/onbloc/go-to-gno-recap-intro-to-the-gno-stack-with-memeland-284a43d7f620) with
+OnBloc, and introduced Gno.land to Go developer communities in Seoul.
+
+## Introduction to Gno.land
+Gno.land Developer Relations Engineer, Leon Hudak, introduced attendees to the 
+project, and explained Gno.land's main features that make it unique:
+
+- The fully open-source smart-contracting platform
+- The custom smart-contracting language of the future (Gno)
+- The new, under-development consensus protocol, Proof of Contribution
+
+[![points](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/thumbs/three-points.png)](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/three-points.png)
+#### Brief overview of gno.land in Japanese
+
+Gno.land's DevRel Engineer, Leon, covered these three topics separately,
+discussing the importance of such features in open-source blockchain systems.
+
+[![leon-poc](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/thumbs/leon-poc.png)](https://gnolang.github.io/blog/2024-04-15_gno-tokyo/src/leon-poc.png)
+#### @leohhhn with the PoC explanation slide in Japanese
+
+Leon delved into the details of Proof of Contribution, and how it connects to the 
```

Comment:
This is a lot of Leon, I would maybe say - the presentation delved into the details...

Reply:
Rephrased it! [c087cd3](https://github.com/gnolang/blog/pull/76/commits/c087cd3c5f7c54534c2066524e02b118f1fdf8be)